exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/registry/RemoteFunctionRegistry.java

```diff
@@ -84,6 +84,7 @@
   private static final String registry_path = "registry";
   private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(RemoteFunctionRegistry.class);
   private static final ObjectMapper mapper = new ObjectMapper().enable(INDENT_OUTPUT);
+  public static final int UNREACHABLE = Integer.MIN_VALUE;
```

Comment:
Maybe better to use long but not int?
Because in that case will be autoconversion in private "boolean isRegistrySyncNeeded(long remoteVersion, long localVersion)"

Just a thought. It is not reccomended in java to compare int and long directly.